packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -195,13 +211,53 @@ export class Autosuggest {
       this.input.addEventListener("input", this.onInput);
       this.input.addEventListener("keydown", this.onKeyDown);
       this.input.addEventListener("focusin", this.onFocusIn);
+
+      window.addEventListener("resize", this.onWindowResize);
+
+      document.addEventListener("scrollend", this.onScrollend);
+
+      this.resizeObserver.observe(this.host);
+
+      this.setListboxContainerMaxBlockSize();
     });
   }
 
   disconnectedCallback() {
     this.input?.removeEventListener("input", this.onInput);
     this.input?.removeEventListener("keydown", this.onKeyDown);
     this.input?.removeEventListener("focusin", this.onFocusIn);
+
+    window.removeEventListener("resize", this.onWindowResize);
+
+    document.removeEventListener("scrollend", this.onScrollend);
+
+    this.resizeObserver?.disconnect();
```

Comment:
Optional chaning (`?.`) is niet nodig, `this.resizeObserver` is nooit `undefined`.